packages/nx-semantic-release/src/common/git.ts

```diff
@@ -33,7 +33,9 @@ export async function isCommitAffectingProjects({
 
   const affectedFiles = await listAffectedFilesInCommit(commit);
   const fileChanges = calculateFileChanges(affectedFiles, [], { projects });
-  const filteredGraph = filterAffected(graph, fileChanges);
+  const filteredGraph = await Promise.resolve(
+    filterAffected(graph, fileChanges)
+  );
```

Comment:
```suggestion
  const filteredGraph = await filterAffected(graph, fileChanges);
```

Wouldn't that work as well? If it won't return a promise, then `await` will just do nothing here and return the value